packages/client/src/views/GrantDetails.vue

```diff
@@ -11,7 +11,7 @@
         <b-row>
 
           <!-- Left page column: title, table data, and grant description -->
-          <b-col>
+          <b-col class="ml-5">
```

Comment:
I think these margins are a bit cleaner to add/maintain if you add them as a `mx-5` on the top-level `section` tag in the component. (And you can remove the `container-fluid` class while you're there... I should have removed it when I did the design touch-ups, as it's redundant. If you're feeling extra, I'm realizing you could also remove the extraneous `div` on L3, since it's purely vestigial.)

Reply:
good call! pushed a new commit for these suggestions. I ended up doing m-5 since having a bottom margin looks nice too.